src/scripts/components/layer/wv.layer.list.js

```diff
@@ -14,7 +14,9 @@ export default class LayerList extends React.Component {
       layerFilter: props.config.layerOrder,
       expandedLayers: [],
       width: props.width,
-      height: props.height
+      height: props.height,
+      metadataLoaded: props.metadataLoaded,
```

Comment:
What do you think about naming this `isMetadataLoaded` to indicate that it's a boolean value? That could help with reading the code, since prefixing a variable name with `is` is a standard naming convention for boolean variables. That way, developers won't have to dig through the code to find where this is defined to know that they're working with a boolean.